bundles/org.openhab.ui/web/src/pages/settings/rules/module-description-suggestions.js

```diff
@@ -12,80 +12,80 @@ export default {
         // triggers
         case 'timer.TimeOfDayTrigger':
           if (!config.time) return moduleType.label
-          return 'When the time is ' + config.time
+          return 'the time is ' + config.time
         case 'timer.GenericCronTrigger':
           if (!config.cronExpression) return moduleType.label
           try {
             return cronstrue.toString(config.cronExpression, {
               use24HourTimeFormat: true
-            })
+            }).toLowerCase()
```

Comment:
This won't work well with labels like "At 08:00, only on Monday and Thursday".
Just lowercase the first character.

Reply:
Sorry I force pushed - habit. But it's now only lowercasing the first letter.